keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExportController.kt

```diff
@@ -87,7 +89,8 @@ class ExportController(
       kind = kind
     )
 
-    return runBlocking {
+    MDC.put("X-SPINNAKER-RESOURCE-ID", "$cloudProvider:$kind:$account:$name")
+    return runBlocking(MDCContext()) {
```

Comment:
Actually, looking at the sample output in the PR description, this is all running in the same thread, so regular MDC should work...